frontend/src/api/doctor.ts

```diff
@@ -0,0 +1,53 @@
+import type { Doctor } from "../components/doctor/DoctorControl";
+const commonRequestOptions = {
+  headers: {
+    Accept: "application/json",
+    "Content-Type": "application/json",
+  },
+};
+
+export const getAllDoctor = async () => {
+  const request = await fetch("/api/admin");
+  const result = await request.json();
+
+  //console.log(result);
+  return result;
```

Comment:
If you type this result, it will make the rest of the app easier to code. This is an array of doctor objects, right? `Doctor[]`